prov/efa/src/rxr/rxr_msg.c

```diff
@@ -109,7 +109,8 @@ ssize_t rxr_msg_post_rtm(struct rxr_ep *rxr_ep, struct rxr_tx_entry *tx_entry)
 						  RXR_MEDIUM_MSGRTM_PKT + tagged, 0);
 	}
 
-	if (efa_both_support_rdma_read(rxr_ep, peer) &&
+	if (tx_entry->total_len >= rxr_env.efa_min_read_msg_size &&
```

Comment:
And for messages smaller than this threshold, `inter_max_medium_message_size` would determine the use of segmentation and the medium msg protocol, right?

Reply:
Yes, exactly.